internal/backends/postgresql/collection.go

```diff
@@ -285,9 +326,9 @@ func (c *collection) Explain(ctx context.Context, params *backends.ExplainParams
 		return nil, lazyerrors.Error(err)
 	}
 
-	return &backends.ExplainResult{
-		QueryPlanner: must.NotFail(types.NewDocument("Plan", queryPlan)),
-	}, nil
+	res.QueryPlanner = queryPlan
+
+	return res, nil
```

Comment:
I think we need to update `backend.ExplainResult` to have SortPushdown field.

Could you ensure to run postgresql integration tests related to sorting for `explain` command, please.